src/tmp.jl

```diff
@@ -240,26 +240,52 @@ function tmp_create_union_tables(connection)
     # These are equivalent to the partitions in time-resolution.jl
     # But computed in a more general context to be used by variables as well
 
+    # Incoming flows
+    DuckDB.execute(
+        connection,
+        "CREATE OR REPLACE TEMP TABLE t_union_in_flows AS
+        SELECT DISTINCT to_asset as asset, year, rep_period, time_block_start, time_block_end
+        FROM flow_time_resolution
+        ",
+    )
+
+    # Outgoing flows
+    DuckDB.execute(
+        connection,
+        "CREATE OR REPLACE TEMP TABLE t_union_out_flows AS
```

Comment:
These new tables are available in the connection object. Having an overview of all tables available in the connection will be nice, including which ones are read from the files/DB/csv and the ones that are created internally in the model.

Reply:
That's the Miro description. When finished it can be rewritten into an official SQL description if desired, to have something in code format.

Reply:
Great, I have created #941 to keep it in our minds.